README.md

```diff
@@ -1,4 +1,4 @@
-# ![Substra](./substra-logo.svg)
+<img src="substra-logo.svg" alt="drawing" width="200"/>
```

Comment:
Could you add the `<h1> ` around the logo again? And change the alt from "drawing" to "substra" ? Thanks!

Reply:
Sure, thank you for this insight @jmorel!
Done